app/code/community/Sailthru/Email/Model/Observer/Adminhtml.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+class Sailthru_Email_Model_Observer_Adminhtml
+{
+    /**
+     * Catch loading of Catalog Product Admin and insert a new mass action to send to Sailthru.
+     * @param Varien_Event_Observer $observer
+     */
+    public function addBlockMassAction(Varien_Event_Observer $observer) {
+        $block = $observer->getEvent()->getBlock();
+
+        // verify this is for a specific store, or a single-store Magento instance
+        $store = $block->getRequest()->getParam('store');
+        if (!$store and count(Mage::app()->getStores()) == 1) {
```

Comment:
Is this the store Id? Is it always 1 if there is only one store.

Reply:
The request param can be null if out of store scope, or -I believe- in single-store mode.